tests/test_dot.py

```diff
@@ -161,6 +161,19 @@ def test_subgraph():
 }'''
 
 
+def test_graph_with_escapes():
+    dot = Graph()
+    dot.node('A', label="\\")
+    dot.node('B', label=r'"\"')
+
+    assert str(dot) == (
```

Comment:
Can you make this more similar to [this](https://github.com/xflr6/graphviz/blob/499875785ccaefe334b6c1dbb45b6eaa41c2a511/tests/test_dot.py#L140) (i.e. use a triple quoted string, raw if needed)?

Reply:
I tried to match that style, but it gets a bit messy. I want to use raw strings otherwise we get '[label="\\\"\\\\\\"]' which an unreadable nightmare. However, if using a raw string then can't use tab escapes. Mixing raw and nonraw strings like this seemed most readable.

If you prefer their being literal tab characters in there, we can change it though.

Reply:
Ah, right the tabs.  Lets leave as is, I might try to adapt after merging.